Increase JVM memory in running tests

In Google Linux environment, the dashboard test fails due to insufficient memory.:

```
~/cloud-opensource-java/dashboard$ mvn install
...
java.lang.OutOfMemoryError: GC overhead limit exceeded
	at java.util.Arrays.copyOfRange(Arrays.java:3664)
	at java.lang.String.<init>(String.java:209)
	at java.lang.StringBuilder.toString(StringBuilder.java:407)
	at sun.net.util.URLUtil.urlNoFragString(URLUtil.java:80)
	at sun.net.www.protocol.jar.JarFileFactory.getCachedJarFile(JarFileFactory.java:122)
	at sun.net.www.protocol.jar.JarFileFactory.get(JarFileFactory.java:81)
	at sun.net.www.protocol.jar.JarURLConnection.connect(JarURLConnection.java:122)
	at sun.net.www.protocol.jar.JarURLConnection.getInputStream(JarURLConnection.java:152)
	at java.net.URLClassLoader.getResourceAsStream(URLClassLoader.java:238)
	at org.apache.bcel.util.ClassPath.getInputStream(ClassPath.java:202)
	at org.apache.bcel.util.ClassPath.getInputStream(ClassPath.java:188)
	at org.apache.bcel.util.MemorySensitiveClassPathRepository.loadClass(MemorySensitiveClassPathRepository.java:95)
	at com.google.cloud.tools.opensource.classpath.ClassDumper.topLevelJavaClassesInJar(ClassDumper.java:359)
	at com.google.cloud.tools.opensource.classpath.ClassDumper.jarFilesToDefinedClasses(ClassDumper.java:323)
	at com.google.cloud.tools.opensource.classpath.ClassDumper.create(ClassDumper.java:90)
	at com.google.cloud.tools.opensource.classpath.StaticLinkageChecker.create(StaticLinkageChecker.java:73)
	at com.google.cloud.tools.opensource.dashboard.DashboardMain.generate(DashboardMain.java:104)
	at com.google.cloud.tools.opensource.dashboard.DashboardTest.setUp(DashboardTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
[ERROR] Tests run: 5, Failures: 0, Errors: 5, Skipped: 0, Time elapsed: 21.4 s <<< FAILURE! - in com.google.cloud.tools.opensource.dashboard.DashboardTest
[ERROR] testCss(com.google.cloud.tools.opensource.dashboard.DashboardTest)  Time elapsed: 0.001 s  <<< ERROR!

```

Increasing `-Xmx1024m` to `-Xmx2048m` solved this issue.

--- a/dashboard/pom.xml
+++ b/dashboard/pom.xml
@@ -88,7 +88,7 @@
           <configuration>
             <!-- To avoid ForkedBooter loading issue -->
             <useSystemClassLoader>false</useSystemClassLoader>
-            <argLine>-Xms128m -Xmx1024m</argLine>
+            <argLine>-Xms128m -Xmx2048m</argLine>
           </configuration>
         </plugin>
     </plugins>

